src/utils/util.js

```diff
@@ -0,0 +1,8 @@
+var Util = {};
+
+// NOTE: Some web-sites override String.prototype.trim method
+Util.trim = function (str) {

```

Comment:
Why it's called `utils.js`? It really bothers me what we have `utils/util`, it doesn't makes sense. Moreover, why it's exports just single function? Why it's not called `string-trim.js` and doesn't `export default` this func. From comment it's seems like it used both by client and server, then why there is no warning comment at the top of the file?